componentified-todo-list/maryam/src/AddNewItem.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {
+  makeStyles,
+  ListItem,
+  IconButton,
+  ListItemSecondaryAction,
+  TextField
+} from '@material-ui/core';
+import {Add, ArrowDownward, ArrowUpward} from '@material-ui/icons'
+// ----------------------------------------------------------------
+const useStyles = makeStyles((theme) => ({
+  form: {
+    '& > *': {
+      margin: theme.spacing(1),
+    },
+  },
+}));
+// ----------------------------------------------------------------
+const AddNewItem = ({todoItems, setTodoItems}) => {
+    const classes = useStyles();
+    let newItem = {};
```

Comment:
__[LOW]__  
Don't use outer scope variables if you can put it inside of the scope itself. It makes code hard to read.